config/environments/staging.rb

```diff
@@ -16,20 +16,20 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { # WARNING do not let standardrb linter change this block, it breaks
-      :address => 'smtp-relay.sendinblue.com',
-      :port => 587,
-      :user_name => ENV["SENDINBLUE_EMAIL"],
-      :password => ENV["SENDINBLUE_PASSWORD"],
-      :authentication => 'login',
-      :enable_starttls_auto => true
+    address: "smtp-relay.sendinblue.com",
```

Comment:
These lines cannot be changed, please see the comment. `# WARNING do not let standardrb linter change this block, it breaks`